src/nimgen/c2nim.nim

```diff
@@ -5,6 +5,14 @@ when (NimMajor, NimMinor, NimPatch) < (0, 19, 9):
 
 import external, file, fileops, gencore, globals
 
+const passCBase = """
+
+import os, strutils
+# import std/time_t  # To use C "time_t" uncomment this line and use time_t.Time
+
```

Comment:
Maybe leverage https://github.com/nimterop/nimterop/blob/master/nimterop/types.nim#L16?

Reply:
On the other hand, this will get written into every single wrapper regardless of whether it needs time or not. Not sure I'm a fan.

Reply:
But Nimgen wont depend on Nimterop.

Reply:
I recommend you NOT to use naked `Time` you will get bitten, see:
https://github.com/nim-lang/zip/issues/47#issue-481984593
Maybe better to use fully qualified.